wandb/wandb_controller.py

```diff
@@ -738,6 +738,43 @@ def _sweep_status(sweep_obj, sweep_conf, sweep_runs):
 
 
 def sweep(sweep, entity=None, project=None):
+    """Register a specified sweep with the W&B backend.
+
+    Parameters
+    ----------
+    sweep : dict, SweepConfig, or callable
+        The sweep configuration (or configuration generator). If a
+        dict or SweepConfig, should conform to the W&B sweep config
+        spec (https://docs.wandb.ai/guides/sweeps/configuration). If a
+        callable, should take no arguments and return a dict that
+        conforms to the W&B sweep config spec.
+    entity: str, optional
+        An entity is a username or team name where you're sending
+        runs. This entity must exist before you can send runs there,
+        so make sure to create your account or team in the UI before
+        starting to log runs.  If you don't specify an entity, the run
+        will be sent to your default entity, which is usually your
+        username. Change your default entity in
+        [Settings](wandb.ai/settings) under "default location to
+        create new projects".
+    project: str, optional 
+        The name of the project where you're sending the new run. If
+        the project is not specified, the run is put in an
+        "Uncategorized" project.
+
+    Returns
+    -------
+    sweep_id : str
+        Unique identifier for the sweep. 
+
+    Examples
+    --------
+
+        >>> sweep_id = wandb.sweep({'name': 'my-awesome-sweep', 'program': 'train.py', 
+        ...                         'metric': 'accuracy', 'method': 'grid', 
+        ...                         'parameters': {'a': {'values': [1, 2, 3, 4]}}})
```

Comment:
This is great! I would add a line beneath that consumes the `sweep_id` with `wandb.agent`, in the simplest way possible, and includes a comment indicating that that's the line that actually kicks off the experiment.

Reply:
done